### DISCUSSION TITLE
i want to know that how to use v-slot instead slot directive in jsx-vue2

### DISCUSSION THREAD

**Discussion by Yh1404** [vuejs/rfcs#494] [General]
<img width="884" alt="image" src="https://user-images.githubusercontent.com/39335644/229449268-94d18d55-519e-4525-89a7-5c6ae7371993.png">

this way has a problem that the table are not rerender when i update array for the table header, and $forceUpdate is not working either

**Comment by posva** [comment]
You can ask questions on the [forum](http://forum.vuejs.org/), the [Discord server](https://chat.vuejs.org/) or [StackOverflow](http://stackoverflow.com/tags/vue.js).
